### ISSUE TITLE
Orthostatic intolerance

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6805]
Preferred term label:
Orthostatic intolerance

Definition (free text, please give PubMed ID)
PMID: 32986636

Parent term (use hpo.jax.org/app)
No found because of the specificity of the disease

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
ORPHA:210272

**Comment by pnrobinson** [comment]
Looking at this article, orthostatic intolerance is. not a characteristic of Mal de débarquement syndrome, but is said to occur with some of the differential diagnoses. This is not a great HPO term because it covers a number of different abnormalities such as orthostatic hypotension (which is already in the HPO).
